data/dotnet/naming.md

```diff
@@ -169,4 +169,4 @@ Localized resources can be referenced as if they were properties, so their namin
 
 ## Assembly and DLL names
 
-Using a name that represents the large chunks of functionality, such as `System.Data`, helps in discoverability.
\ No newline at end of file
+Using a name that represents the large chunks of functionality, such as `System.Data`, helps in discoverability.
```

Comment:
Not sure what's going on here. I used the web editor. Perhaps it added a newline where there wasn't one? I can clone and edit in a proper editor if this is a big deal.